nameko/events.py

```diff
@@ -197,19 +235,26 @@ def setup(self):
             queue_name = "evt-{}-{}".format(self.source_service,
                                             self.event_type)
         elif self.handler_type is BROADCAST:
+            broadcast_identifier = self.broadcast_identifier
             queue_name = "evt-{}-{}--{}.{}-{}".format(self.source_service,
                                                       self.event_type,
                                                       service_name,
                                                       self.method_name,
-                                                      uuid.uuid4().hex)
+                                                      broadcast_identifier)
 
         exchange = get_event_exchange(self.source_service)
 
-        # auto-delete queues if events are not reliably delivered
+        # queues are auto-deleted unless reliable delivery is requested
```

Comment:
this comment (and the one for `exclusive`) don't add much info beyond the line of code they decorate. could we change it to explain _why_ we want this behaviour (or drop, though maybe at least include the point about `exclusive` silently overwriting the `auto_delete` behaviour)